nhs/data/handling.py

```diff
@@ -103,4 +106,68 @@ def read_spreadsheets(
             lambda x: x[0], placeholder_matches(files, file_dir_pattern, ["key"])
         )
 
-    return {key: val for key, val in zip(keys, map(reader, files))}
+    return {key: reader(file, sheet_id) for key, file in zip(keys, files)} if extension == "xlsx" else {
+        key: reader(file) for key, file in zip(keys, files)
+    }
+
+
+@log_entry_exit(level="INFO")
+def standarise_names(df_dict: dict[str, pl.LazyFrame | None], census_metadata: pl.LazyFrame | None,
+                     identification: str, abbreviation_column_name: str,
+                     long_column_name: str) -> dict[str, pl.LazyFrame | None]:
+    """
+    Process the column names of a polar Lazy frame dictionary to make them more readable.
+        1. Expand abbreviated names.
+        2. Converting to lower case.
+    If any parameters are None, this function returns prev_csv_pl_df without change.
+    The filename in both dictionaries are checked or filtered to be passed.
+
+    Parameters:
+    --------
+    param1 : dict[str, pl.LazyFrame]
+        df_dict: A dictionary containing census data as value and filename as key or None.
+    param2 : [pl.LazyFrame | None]
+        census_metadata: Census metadata lazy frame with one standard sheet or None.
+        Blank files won't change prev_csv_pl_df.
+    param3 : str
+        identification: Identify different files in df_dict. Could not be None
```

Comment:
Unclear, what should these `identification`s be? are they codes like "G01"? It's also a string, what's the purpose of this? Can also remove `Could not be None` because that's written in the type signature of `str`

Reply:
rename to `census_code_col`, change docstring to: name of the column in `census_metadata` that contains names in the `df_dict` to standarise. e.g. if df_dict have the pair "G03": <LazyFrame> and "G03" exist in the `census_code_col` column in metada, then the lazy frame will be standarised